src/poly/polymorphisem/patrick.java

```diff
@@ -1,4 +1,4 @@
-package polymorphisem;
+package poly.polymorphisem;
 
 public class patrick extends Fish implements BasketballPlayer {
```

Comment:
```suggestion
public class Patrick extends Fish implements BasketballPlayer {
```
class names begin with capital letters